### PR TITLE
fix (gql-middleware): Prevents resending all subscriptions when a user leaves a meeting

### PR DESCRIPTION
**Pull request by gustavotrott**
**TL;DR:** This PR prevents resending all subscriptions when a user leaves a meeting, allowing only the PresenceManager subscriptions to be re-sent.

**Details:**

Currently, when a user changes their role or another important status, the GraphQL-Middleware reconnects with Hasura and re-sends all subscriptions. This is expected behavior when a user remains in the meeting.

When a user leaves the meeting, this reconnection and retransmission still occur. 
The issue arises because after leaving, the user loses permission to query most Graphql Types (tables). As a result, re-sending these subscriptions cause permission errors.

This PR addresses the issue by filtering the subscriptions. When the `Hasura-Role` is different from `bbb_client`, only the PresenceManager subscriptions will be re-sent. PresenceManager queries are allowed for all users, including those with the `not_joined_bbb_client` role.

**Closes:** #21026

### CODE REVIEW

**Review by JoVictorNunes — APPROVED** [review]
Can't reproduce the bug! LGTM!
